src/core/reference/src/runtime/reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -276,6 +277,14 @@ void ROIAlignForward(const int64_t nthreads,
                 }  // for pw
             }      // for ph
         }          // for c
+    };
```

Comment:
Do you have any performance experiments on real models if it improves performance?